packages/helm-charts/oracle/USDCUSD.yaml

```diff
@@ -16,14 +16,19 @@ oracle:
         {exchange: 'KRAKEN', symbol: 'USDCUSD', toInvert: false}
       ],
       [
-        {exchange: 'BITSTAMP', symbol: 'USDCUSD', toInvert: false}
+        {exchange: 'BINANCEUS', symbol: 'USDCUSD', toInvert: false}
       ],
       [
-        { exchange: 'GEMINI', symbol: 'USDCUSD', toInvert: false }
+        {exchange: 'COINBASE', symbol: 'USDTUSDC', toInvert: true},
+        {exchange: 'COINBASE', symbol: 'USDTUSD', toInvert: false}
+      ],
+      [
+        {exchange: 'WhiteBIT', symbol: 'USDCUSDT', toInvert: false},
+        {exchange: 'COINBASE', symbol: 'USDTUSD', toInvert: false}
       ]
     ]"
```

Comment:
Interesting that OKX and KuCoin rates are more unstable although a higher volume seems to be traded there

Reply:
Rates and config look good to me :)

Reply:
Yes, they were my initial candidates. But maybe WhiteBIT has better merket makers...